client/web/src/user/settings/auth/ExternalAccountsSignIn.tsx

```diff
@@ -65,35 +79,63 @@ const getNormalizedAccount = (
     // if external account exists - add user specific data to normalizedAccount
     if (account && accountExternalData) {
         switch (kind) {
-            case 'GITHUB':
-                {
-                    const githubExternalData = accountExternalData as GitHubExternalData
-                    normalizedAccount = {
-                        ...normalizedAccount,
-                        external: {
-                            id: account.id,
-                            // map github fields
-                            userName: githubExternalData.name,
-                            userLogin: githubExternalData.login,
-                            userUrl: githubExternalData.html_url,
-                        },
-                    }
+            case 'github': {
+                const githubExternalData = accountExternalData as GitHubExternalData
+                normalizedAccount = {
+                    ...normalizedAccount,
+                    external: {
+                        id: account.id,
+                        // map GitHub fields
+                        userName: githubExternalData.name,
+                        userLogin: githubExternalData.login,
+                        userUrl: githubExternalData.html_url,
+                    },
                 }
+            }
                 break
-            case 'GITLAB':
-                {
-                    const gitlabExternalData = accountExternalData as GitLabExternalData
-                    normalizedAccount = {
-                        ...normalizedAccount,
-                        external: {
-                            id: account.id,
-                            // map gitlab fields
-                            userName: gitlabExternalData.name,
-                            userLogin: gitlabExternalData.username,
-                            userUrl: gitlabExternalData.web_url,
-                        },
+            case 'gitlab': {
+                const gitlabExternalData = accountExternalData as GitLabExternalData
+                normalizedAccount = {
+                    ...normalizedAccount,
+                    external: {
+                        id: account.id,
+                        // map gitlab fields
+                        userName: gitlabExternalData.name,
+                        userLogin: gitlabExternalData.username,
+                        userUrl: gitlabExternalData.web_url,
+                    },
+                }
+            }
+                break
+            case 'saml': {
+                // The data stored in user_external_account has this structure: “Values”[“<attribute names for email or username>"] > Values[0] > “Value”: string
```

Comment:
```suggestion
                // The data stored in user_external_account has this structure: "Values"["<attribute names for email or username>"] > Values[0] > "Value": string
```

There's a mix of different kinds of quotes here